ratatui/Cargo.toml

```diff
@@ -126,7 +123,7 @@ palette = ["ratatui-core/palette", "dep:palette"]
 
 ## Use terminal scrolling regions to make some operations less prone to
 ## flickering. (i.e. Terminal::insert_before).
-scrolling-regions = []
+scrolling-regions = ["ratatui-core/scrolling-regions"]
```

Comment:
Do we also need to add `"ratatui-crossterm?/scrolling-regions` etc. here?

https://doc.rust-lang.org/cargo/reference/features.html#dependency-features
>The "package-name/feature-name" syntax will also enable package-name if it is an optional dependency. Often this is not what you want. You can add a ? as in "package-name?/feature-name" which will only enable the given feature if something else enables the optional dependency.

Reply:
da148bf80..29729ed20